lib/components/admin/call-record.js

```diff
@@ -14,6 +15,20 @@ export default class CallRecord extends Component {
     expanded: false
   }
 
+  _getCallDuration = () => {
+    const {call} = this.props
+    const startTimeMoment = moment(call.startTime)
+    const endTimeMoment = moment(call.endTime)
+    const [hours, minutes, seconds] = moment.utc(
+      moment.duration(endTimeMoment.diff(startTimeMoment)).asMilliseconds()
+    ).format('HH:mm:ss').split(':').map(val => +val)
+    const parts = []
+    if (hours) parts.push(`${hours} hr`)
+    if (minutes) parts.push(`${minutes} min`)
+    if (seconds) parts.push(`${seconds} sec`)
+    return parts.join(', ')
```

Comment:
In datatools-ui we use [humanize-duration](https://www.npmjs.com/package/humanize-duration). Should we use it here?